docs/application/dotnet/guides/text-input/voice-control-manager.md

```diff
@@ -0,0 +1,1096 @@
+        -   The current language can be changed by setting application or by changing the language on the device display.
+    -   Supported language
+        -   You can retrieve a list of supported languages to check whether the language that you want is supported.
+
+
+To use the voice control manager, follow these steps:
+
+1.  Create a handle and [register event handlers](#callback).
+
+    The initialization allows the voice control to distinguish your application from any other application that also uses voice control. Only one voice control manager instance can work on the device. Therefore, if other application including voice control manager exists, your application cannot work properly on one same device.
+
+    The registered callbacks can be notified of the changes in service status, current language, recognition results, and errors.
+
+2.  Set commands.
+
+    You can create a command list, and add or remove individual commands in the list. While creating an individual command, set the command text and type for each command handle. When all the commands are created and added to the command list, set the command list to the voice control manager for recognition.
+
+3.  Prepare the voice control manager.
+
+    The preparation connects the voice control service for background work, such as recording and recognition the voice.
+
+    When the application initializes and prepares the voice control manager, the voice control service is invoked and connected for the background work. The service and the application communicate as server and client.
+
+4.  Get the recognized command result and ASR.
+
+    The recognized command result and ASR are sent through a registered callback.
+    You can receive the command matching result that the user uttered.
+    Multiple recognition results can happen if the duplicated commands are registered or you request multiple commands. In this case, you can select or reject the results using the `VoiceControlManagerClient.SetRecognizedCommandsSelectionDelegate()` callback.
+
+5.  When no longer needed, unset the voice control manager.
+
+    You must disconnect the voice control service and deinitialize the voice control manager using the `Unprepare()` and `Deinitialize()` methods of the [Tizen.Uix.VoiceControlManager](https://samsung.github.io/TizenFX/latest/api/Tizen.Uix.VoiceControlManager.html) class.
+
+The figure is the overall configuration for voice control framework:
+
+![Voice control manager framework](./media/csapi_voice_control_framework.png)
+
+The VC-Manager(voice control manager) can cancel, start or stop the VC-EngineService(voice control engine service) voice recording. And the recognized result is received from VC-EngineService. The VC-EngineService analyzes voice recording data and deliver the perceived result to the manager. The VC-Client(voice control client) can register commands to recognize and register them in each application.
```

Comment:
@sungraejo Please avoid using short-forms for voice control manager, voice control engine service, etc. to maintain consistency on the page. The paragraph is confusing for an end-user to read. Especially "And the recognized result is received from VC-EngineService." this part, seems incomplete as it doesn't define why this line is mentioned. You can elaborate more on this to avoid any confusion. Whereas what I am understanding is the following:

The voice control manager can cancel, start, or stop the voice control engine service voice recording. Additionally, the recognized result is received from the voice control engine service then the voice control engine service analyzes voice recording data and delivers the perceived result to the manager. Then voice control clients can register commands to recognize and register them in each application.

Please correct me if I am wrong. Feel free to explain more so I can rephrase better.